### ISSUE TITLE
[FEATURE] Dependencies should be updated to dedupe the tree for npm

### ISSUE THREAD

**Issue by stof** [npm/gauge#127]
# What / Why

The `npm` package depends on `cli-table3`, which uses `string-width` and `strip-ansi` in recent versions.
It also uses this `gauge` package (through `npmlog`) which uses older versions of `string-width` and `strip-ansi`. the `wide-align` dependency also brings its own versions of `string-width` and `strip-ansi`, which are potentially different too.

## When

* n/a

## Where
* n/a

### Current Behavior
Packages `string-width` and `strip-ansi` (and potentially others too, but I stopped analyzing the tree at some point) are installed many times in different versions

### Expected Behavior
The dependencies should be updated to use recent versions, so that more deduplication can be done (and the maintenance status of older major version could be another reason to upgrade).

## Who
* n/a

## References
* n/a

**Comment by trmpowell** [comment]
On top of this, Snyk reports a high severity vulnerability from `strip-ansi@4.0.0`
```
  ✗ Regular Expression Denial of Service (ReDoS) [High Severity][https://snyk.io/vuln/SNYK-JS-ANSIREGEX-1583908] in ansi-regex@3.0.0
    introduced by gauge@3.0.1 > strip-ansi@4.0.0 > ansi-regex@3.0.0 and 2 other path(s)
  This issue was fixed in versions: 6.0.1
```

Upgrade `strip-ansi` to version 7.0.1+ to resolve this.

**Comment by wraithgar** [comment]
https://github.com/npm/gauge/pull/138
